src/textures/mlx_xpm42.c

```diff
@@ -37,6 +37,17 @@
  * straight forward to this format however.
  */
 
+/**
+ * Parses HEX color channel e.g: "0F"
+ * 
+ * @param channel The 2 character string to parse.
+ * @return Int value of the channel.
+ */
+static uint8_t parse_hex_channel(char *channel) {
+	char temp_chan[] = {channel[0], channel[1], '\0'};
+	return (uint8_t)strtol(temp_chan, NULL, 16);
+}
+
 /**
  * Parses the XPM coolor value entry e.g: ".X #00FF00FF"
```

Comment:
Just noticed this smoll typo, feel free to fix this, or not.

Reply:
Addressed in: 446ef18